classes/decision/types/traits/RequestPayment.php

```diff
@@ -78,7 +78,6 @@ protected function requestPayment(Submission $submission, User $editor, Context
         $authorIds = $this->getAssignedAuthorIds($submission);
         foreach ($authorIds as $authorId) {
             $notificationMgr->createNotification(
-                Application::get()->getRequest(),
```

Comment:
@touhidurabir, are you sure about this change?

Reply:
Ah, I see this is removed further up the stack because it wasn't used. Never mind.